src/pykiso/lib/auxiliaries/udsaux/uds_auxiliary.py

```diff
@@ -73,13 +75,16 @@ def send_uds_raw(
             log.info(
                 f"UDS request to send '{['0x{:02X}'.format(i) for i in msg_to_send]}'"
             )
-            resp = self.uds_config.send(msg_to_send)
+            resp = self.uds_config.send(msg_to_send, responseRequired=response_required)
```

Comment:
https://github.com/richClubb/python-uds/blob/master/uds/uds_communications/Uds/Uds.py#L142 kind of shocked me! So how many ticks do we wait for it to return?

Reply:
In this case we don't wait, if a response is required, the default value is either 1s, which seems a lot, or 1ms (the comment and the variable name are contradictory) : https://github.com/eclipse/kiso-testing-python-uds/blob/master/uds/uds_communications/TransportProtocols/Can/CanTp.py#L251